Documentation/devel/architecture.md

```diff
@@ -2,19 +2,25 @@
 
 ## Overview
 
-rkt consists only of a command-line tool, `rkt`, and does not have a daemon. This architecture allows rkt to be updated in-place without affecting application containers which are currently running. It also means that levels of privilege can be separated out between different operations.
+rkt consists only of a command-line tool, `rkt`, and does not have a daemon.
```

Comment:
Hm, not sure what to do about it. I guess that API service doesn't make this paragraph obsolete.